### PR TITLE
Add project start guide

### PR DESCRIPTION
**Pull request by yuri-sergiichuk**
In this PR I created a guide that goes over the steps of starting a new project, covered the common development flow to be used while developing projects.

### CODE REVIEW

**Comment by yuri-sergiichuk** [comment]
@alexander-yevsyukov, @armiol PTAL.

**Comment by yuri-sergiichuk** [comment]
@alexander-yevsyukov @oleksandra-kovalenko PTAL.

File: docs/guides/start-new-project.md [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+---
+
+# Starting a new project
+
+<p class="lead">Identifying a particular development flow may be challenging at times. 
+Making the flow efficient and straight-forward is even harder. This guide will walk you through 
+the way we tend to start and develop new projects.</p>
+
+While the development process is described in the [Introduction][introduction] section, 
+in this guide we want to cover the steps that we follow while working on a project in more detail.
```
Comment (oleksandra-kovalenko) [inline docs/guides/start-new-project.md:15]:
```suggestion
While the key stages of the development process are described in the [Introduction][introduction] section, this guide provides more hands-on details on the steps that we follow while working on a project.
```

File: docs/guides/start-new-project.md [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+headline: Documentation
+bodyclass: docs
+layout: docs
+---
+
+# Starting a new project
+
+<p class="lead">Identifying a particular development flow may be challenging at times. 
+Making the flow efficient and straight-forward is even harder. This guide will walk you through 
+the way we tend to start and develop new projects.</p>
```
Comment (oleksandra-kovalenko) [inline docs/guides/start-new-project.md:12]:
```suggestion
<p class="lead">Starting off a new project goes smoother when you have an efficient and straightforward development flow to rely on. In this guide, we describe how we usually start and develop new projects</p>
```

File: docs/guides/start-new-project.md [hunk 3]
```diff
@@ -0,0 +1,259 @@
...
+<p class="note">Please consult with the Naming Conventions [guide][identifiers-naming] for our 
+recommendations on naming the identifier types.</p>
+
+When the ID types are defined, please create a Pull Request so that the team can review 
+and polish the code of this important development step.
+
+### Events
+
+When the IDs are defined it’s time to define [event][event-concept] messages. The events are named 
+as facts formulated as past participles. They are defined in files with the
```
[the remark this answers is not in the document]
Comment (yuri-sergiichuk) [reply docs/guides/start-new-project.md]: Done.

File: docs/guides/start-new-project.md [hunk 4]
```diff
@@ -0,0 +1,259 @@
...
+either a UI if one is needed, or the public API, or a dedicated idiomatic client.
+
+As soon as the implementation is ready, another PR and review come along.
+
+## Start over again
+
+When the scenario is finished, pick up a new one and start it over again following the PR 
+and review process.
+
+As soon as the Bounded Context is fulfilled the team can continue with the next one.
```
[the remark this answers is not in the document]
Comment (yuri-sergiichuk) [reply docs/guides/start-new-project.md:219]: I don't think it's a good idea to reference the steps here as they are pretty far from this sentence.

**Pull request merged**